### PR TITLE
Issue 59

### PR DESCRIPTION
**Pull request by CSteinmuller**
Added ini for database config data and code for reading said ini. Default ini contains data originally found in code.

### CODE REVIEW

File: app/Core/Database.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Core;
-
+$config_data = parse_ini_file("dbconfig.ini");
+print_r($ini_array);
```
Comment (mluzarowDEP) [inline app/Core/Database.php]: Debug print.

File: app/Core/Database.php [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Core;
-
+$config_data = parse_ini_file("dbconfig.ini");
```
Comment (mluzarowDEP) [inline app/Core/Database.php]: This shouldn't be in Core. It should be in the root directory (the some place where the index is).

Comment (mluzarowDEP) [inline app/Core/Database.php]: This should also be a part of the initialize method.

File: app/Core/Database.php [hunk 3]
```diff
@@ -29,7 +30,7 @@ public static function getConnectionData () {
 	 * @return bool database status flag
 	 */
 	public static function initialize () : bool {
-		self::$database = new \mysqli ('localhost:3306', 'root', 'glitch123');
+		self::$database = new \mysqli ($config_data['location'], $config_data['user'], $config_data['password']);
```
Comment (mluzarowDEP) [inline app/Core/Database.php]: 3306 is the chosen MariaDB port when setting up the db. This and the actual db location (in this self hosted case it is localhost but should be changeable depending on how the user wants to set up this server. These should be two different variables.

File: app/Core/Database.php (hunk 1 above)
Comment (CSteinmuller) [reply app/Core/Database.php]: fixed

File: app/Core/Database.php (hunk 2 above)
Comment (CSteinmuller) [reply app/Core/Database.php]: Moved

File: app/Core/Database.php (hunk 3 above)
Comment (CSteinmuller) [reply app/Core/Database.php]: fixed

File: app/Core/Database.php (hunk 2 above)
Comment (CSteinmuller) [reply app/Core/Database.php]: The ini file itself?

**Pull request closed without merging**